sgf-viewer/src/components/subcomponents/GameList.svelte

```diff
@@ -14,13 +13,49 @@
         num_moves: "Game length",
         sgf_path: "SGF Path", // Should also include line number
     };
-    function indexToSgfPath(index: number) {
-        let keyIndex = Object.keys(tableColumns).indexOf("sgf_path");
-        let fileName = games[index][keyIndex].split("/").slice(-1)[0]; // Get last element
+    const sgfPathKeyIndex = Object.keys(tableColumns).indexOf("sgf_path");
+    $: sgfPaths = games.map((game) => {
+        const fileName = game[sgfPathKeyIndex].split("/").slice(-1)[0]; // Get last element
         return `/sgfs/${dirName}/${fileName}`;
+    })
+
+    const wgoPlayer = getContext("wgoPlayer");
+    const updateGame = getContext("updateGame");
+    let selectedRow: number = 0;
+    // Current game loaded by the wgoPlayer. Doesn't necessarily match with
+    // selectedRow if wgoPlayer is not yet initialized.
+    let currentSgfPath: string = "";
+    // Arbitrary large number, should be be larger than the move count in any of
+    // our SGFs. This is used to load the last move of the game.
+    const LAST_MOVE_NUM = 10000;
+    // Updates wgoPlayer and selectedRow.
+    $updateGame = (row, move = LAST_MOVE_NUM) => {
```

Comment:
this function is now the sole way to update the game board.

Formerly, we updated the game board by updating selectedRow, which updates `sgfPath`, which is passed to sibling component `<GameBoard>` via two-way binding through shared parent `<Section>`. And also move links would call a global function defined by `<GameBoard>` to update the move number